book/contributions/rec_using_inchi/recipes/rec_using_inchi.md

```diff
@@ -0,0 +1,107 @@
+# Using InChI and InChIKey to annotate your chemical entities 
+
+```{dropdown} About this interactive ![icons](../static/img/rocket.png) recipe
+- Author: [Jordi Cuadros](https://orcid.org/0000-0001-6513-9140)
+- Reviewer:
+- Topics: InChI, InChIKey
+- Format: Tutorial
+- Scenarios: Retrieve the InChI and InChIKey of a chemical
+- Skills:
+    - Understand what an InChI is (<http://publications.iupac.org/ci/2006/2806/2806-pp12-15.pdf>, <https://www.inchi-trust.org/>)
+    - Know what MOL files are and where to get them (<https://en.wikipedia.org/wiki/Chemical_table_file>, <https://discover.3ds.com/sites/default/files/2020-08/biovia_ctfileformats_2020.pdf> )
+    - Understand what a SMILES string is (<https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system>, <http://opensmiles.org/>)
+- Learning outcomes: After completing this example you should understand:
+    - What makes InChI and InChIKey convenient identifier for chemical
+    - How to obtain the InChI and the InChIKey of a chemical
+    - How to identify a chemical behind an InChI or an InChIKey
+- Citation: 'Using InChI and InChIKey to annotate your chemical entities', The IUPAC FAIR Chemistry Cookbook, https://iupac.github.io/WFChemCookbook/recipes/rec_using_inchi.html
+- Reuse: This notebook is made available under a [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) license.
+```
+
```

Comment:
General comment that the text of each section should be on multiple lines to make it easy to read when doing a PR.